scripts/workflow.rb

```diff
@@ -83,9 +50,53 @@ def release
       File.delete(filename)
     end
 
-    set_released_flag_for_completed_transcripts if @mode == 'COMPLETED'
+    set_released_flag_for_completed_transcripts(release_needin_ids) if @completed
+
     puts 'Ah... That is the stuff.'
   end
+
+  private
+
+  def create_guid_file(guids)
+    # write guids to file
+    File.open("ready_for_release_guids_#{ @right_now }.txt", "w+") do |f|
+      guids.each do |guid|
+        f << guid + "\n"
+      end
+    end
+  end
+
+  def get_from_release_count_json
+    puts "Alright, first we're going to have to ask #{TX_HOST} which ones are ready to release..."
+    response = RestClient.get(TX_HOST + '/release_count.json')
+    raise 'Didnt get a response for release_count...' unless response.body
+    JSON.parse(response.body)['data']
+  end
+
+  def get_from_all_uids_json
+    puts "Alright, first we're going to have to ask #{TX_HOST} for all the Transcript IDs..."
+    response = RestClient.get(TX_HOST + '/all_uids.json')
+    raise 'Didnt get a response for all_uids...' unless response.body
+    JSON.parse(response.body)['data']
+  end
+
+  def set_released_flag_for_completed_transcripts(guids)
+    guids.each do |id|
+      puts "Updating the released flag on: #{id}"
+      RestClient.patch(TX_HOST + '/transcripts/' + id, { 'released' => true } )
+    end
+  end
 end
 
-TranscriptReleaser.new(ARGV).release
+options = {}
+OptionParser.new do |opts|
+  opts.banner = "Usage: example.rb [options]"
+  opts.on("-c", "--complete", "Release completed, but unreleased, transcripts to S3") do |c|
+    options[:completed] = c
+  end
+  opts.on("-a", "--all", "Release ALL the transcripts to S3 with blatant disregard for completeness") do |a|
+    options[:all] = a
+  end
+end.parse!
```

Comment:
What are the values of `c` and `a` in the blocks if the script is invoked like so...
```
bundle exec ruby workflow.rb --all --complete
```

Reply:
If the values passed to the blocks are `true`, then great, but I'm afraid they might be `null`.

Reply:
Just read the docs and your code looks correct. Approving!